dark_emulator/darkemu/cosmo_util.py

```diff
@@ -144,7 +143,9 @@ def f_from_z(self, z):
         return self.f_from_a(a)
 
     def f_from_a(self, a):
-        return derivative(lambda t: a*np.log(self.Dgrowth_from_a(t)), a, dx=1e-6)
+        da = 1e-3
```

Comment:
Would it be safer to use [scipy.differentiate.derivative](https://docs.scipy.org/doc/scipy/reference/generated/scipy.differentiate.derivative.html)?